manual/2-populate-the-structure/canonical-examples/quickstart-example.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Quickstart for GitHub Repositories
+topic: Repositories
+author: jorge-campo, cheny0
+version: 1
+url: docs/repositories/quickstart
+---
+
+<!-- This document is an example for the concept template. It is not a real document and should be used for illustration purposes only. -->
+
+# Quickstart for GitHub Repositories
+
+#### Create and manage your first repository quickly.
```

Comment:
I understand you're using an existing GitHub's quickstart for this example, but you should adapt it to fulfill our writing rules. We don't use "quickly" (or similar: "faster", "quick", etc.), for example, becase we don't want to assume how fast or slow our users are.